app/Ticket.php

```diff
@@ -155,8 +155,8 @@ public function addComment($user, $body, $newStatus = null)
         }
         $previousStatus = $this->updateStatusFromComment($user, $newStatus);
         $this->associateUserIfNecessary($user);
-        if (! $body) {
-            return;
+        if (! $body or $body === $user->settings->tickets_signature) {
```

Comment:
I think the &#13 chars will be in the $body so the === will fail,

is that correct? or does it really work this way?

Reply:
Seems to ignore it, feel free to test yourself, I only did some quick checks.